### ISSUE TITLE
Temperature data Deep Pond

### ISSUE THREAD

**Issue by sluning** [nickmckay/LiPDverse#132]
I was looking for the temperature data of Deep Pond (Deep.Marsicek.2013) which was published by Shuman et al. 2019: https://www.biorxiv.org/content/10.1101/548420v1
Would it be possible to add it to this site?
Cheers, Sebastian

**Comment by nickmckay** [comment]
Yes. That would be great. Do you have a copy of the data, or can you point
use to where it's archived online?

**Comment by sluning** [comment]
Hi Nick. No, unfortunately I only have the digitized version. It would be great if Marsicekt et al. 2013 could provide the original data... Cheers, Sebastian
